_posts/2024-06-20-vector-radial-search.md

```diff
@@ -0,0 +1,360 @@
+---
+layout: post
+title:  Understanding Vector Radial Search in OpenSearch
+authors:
+  - junqiu
+  - vamshin
+  - dylantong
+date: 2024-06-20
+categories:
+    - technical-posts
+meta_keywords: 
+meta_description: 
+has_science_table: true
+featured_blog_post: true 
+featured_image: false # /assets/media/blog-images/__example__image__name.jpg
+---
+
+Before OpenSearch version 2.14, the OpenSearch k-NN plugin only provided top-k queries for approximate vector similarity search. OpenSearch 2.14 introduced a new type of vector search---_radial search_. Radial search is now supported in both [k-NN search]((https://opensearch.org/docs/latest/search-plugins/knn/radial-search-knn/)) and [neural search](https://opensearch.org/docs/latest/query-dsl/specialized/neural/) also support radial search.
+
+Radial search enhances the capabilities of the OpenSearch k-NN plugin beyond approximate top-k searches. With radial search, you can search for all points in a vector space that are within a specified maximum distance or minimum score threshold from a query point. This provides increased flexibility and utility in search operations. 
+
+For instance, if you want to search for documents most similar to a given document, you can run a query to retrieve the top K most similar documents (where K = top N). However, if K is not chosen optimally, you might receive irrelevant results or miss relevant ones. With vector radial search, you can run a query to retrieve all highly similar documents by using a threshold (e.g., similarity score > 0.95).
```

Comment:
Above we use "top-k", but here we use "top K" and "K".

Reply:
Changed to "top K"